scriptmodules/supplementary/skyscraper.sh

```diff
@@ -44,14 +45,19 @@ function install_skyscraper() {
         'artwork.xml.example2'
         'artwork.xml.example3'
         'artwork.xml.example4'
+        'platforms.json'
+        'screenscraper.json'
         'tgdb_developers.json'
         'tgdb_publishers.json'
         'mameMap.csv'
         'aliasMap.csv'
```

Comment:
Please add mobygames.json

Reply:
Done, should that be added to the [unix:examples.files](https://github.com/Gemba/skyscraper/blob/7c4677172198d39293e8d74827fd22eb9a0f724c/skyscraper.pro#L14) also?